src/main/resources/application-nomis-hsqldb.yml

```diff
@@ -1,6 +1,6 @@
 spring:
   datasource:
-    url: jdbc:hsqldb:mem:nomis-db;sql.syntax_ora=true;get_column_name=false;shutdown=false;sql.nulls_first=false;sql.nulls_order=false;sql.enforce_refs=true
+    url: jdbc:hsqldb:mem:nomis-db;sql.syntax_ora=true;get_column_name=false;shutdown=false;sql.nulls_first=false;sql.nulls_order=false;sql.enforce_refs=true;sql.concat_nulls=false
```

Comment:
This makes HSQLDB behave like Oracle in that concat('something', null) == 'something'. See the [HSQLDB docs](https://hsqldb.org/doc/2.0/guide/builtinfunctions-chapt.html) (and search for `concat_nulls`).